### PR TITLE
Adjust session layout

### LINKED ISSUE #442: Apply session item design in timeline screens
## Kind (Required)

- Improvement

 ## Overview (Required)

- Please remove button of session's feedback.
- Please add session time left of hall name.
- etc...

 ## Links	

- Figma
    - https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=4%3A2

## Screenshot
Currently | Design
:--: | :--:
 |

### PR DESCRIPTION
**Pull request by a25max**
## Issue
- close #442

## Overview (Required)
- Remove survey
- add session time left of hall name.

## Links
- https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=4%3A2

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionItem.kt [hunk 1]
```diff
@@ -69,7 +69,7 @@ class SessionItem @AssistedInject constructor(
             live.isVisible = session.isOnGoing
             title.text = session.title.ja
             room.text = session.room.name.getByLang(defaultLang())
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionItem.kt]: Can we delete this? 🙏

**Comment by takahirom** [comment]
Can you merge from master? Probably your CI errors are fixed 🙏

**Comment by a25max** [comment]
I see.

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionItem.kt (hunk 1 above)
Comment (a25max) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/item/SessionItem.kt]: I forget that. OK

**Comment by takahirom** [comment]
You may need to do something like this. 🙏 
git pull git@github.com:DroidKaigi/conference-app-2020 master

**Comment by takahirom** [comment]
Thanks! LGTM 👍

**Pull request merged**